agent/ecscni/plugin.go

```diff
@@ -111,18 +114,34 @@ func (client *cniClient) setupNS(cfg *Config) (*current.Result, error) {
 	}
 
 	seelog.Debugf("[ECSCNI] Starting ENI (%s) setup in the the container namespace: %s", cfg.ENIID, cfg.ContainerID)
-	os.Setenv("ECS_CNI_LOGLEVEL", logger.GetLevel())
-	defer os.Unsetenv("ECS_CNI_LOGLEVEL")
 
-	// Invoke eni plugin ADD command
-	result, err := client.add(runtimeConfig, cfg, client.createENINetworkConfig)
-	if err != nil {
-		return nil, errors.Wrap(err, "cni setup: invoke eni plugin failed")
+	// Invoke eni plugin ADD command based on the type of eni plugin
+	if cfg.ENIType == apieni.BranchENIType {
+		os.Setenv("VPC_CNI_LOG_LEVEL", logger.GetLevel())
+		os.Setenv("VPC_CNI_LOG_FILE", vpcCNIPluginPath)
```

Comment:
appmesh plugin got around this environment variable by [here](https://github.com/aws/amazon-vpc-cni-plugins/blob/master/plugins/aws-appmesh/plugin/plugin.go#L27)

Reply:
Discussed offline, going forward with the environment variable way of setting the log path for now.